service/src/main/resources/application.yml

```diff
@@ -21,12 +21,6 @@ env:
 
 # Below here is non-deployment-specific
 
-# When the target is 'local' the write-config.sh script will generate this properties file. It
-# contains the configuration of the BPM test application. We can use that application in our
-# integration testing to make sure the application code paths are working. However, we do not
-# want it to appear in production environments.
-spring.config.import: optional:file:../config/local-properties.yml
```

Comment:
This change isn't directly related, but I noticed it and we don't use `local-properties.yml`. The `write-config.sh` script it refers to doesn't exist in this project either.